dist/docker/debian/build_docker.sh

```diff
@@ -26,7 +26,13 @@ version="$(<build/SCYLLA-VERSION-FILE)"
 release="$(<build/SCYLLA-RELEASE-FILE)"
 
 mode="release"
-arch="$(uname -m)"
+
+if uname -m | grep x86_64 ; then
+  arch="amd64"
+else
```

Comment:
Actually, you need it because debian arch names are different from what uname -a returns.